.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
```

Comment:
I'd add a `workflow_dispacth` trigger here (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) to allow you to run this whenever you want, without waiting for a PR (very, very useful for debugging, unless you like starting up draft PRs just to see if your stuff works)

Reply:
done, thanks